### PR TITLE
Update Kubeless and monitoring test images

### PR DESCRIPTION
**Pull request by gchbg**
The images use the latest stable Alpine Linux as base and automatically
download the latest stable kubectl and kubeless.

Confirm these statements before you submit your pull request:

- [x] I have read and submitted the required [Contributor Licence Agreements](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#agreements-and-licenses).
- [x] This pull request follows the contributing guidelines, recommended Git workflow, and templates.
- [x] I have tested my changes.
- [x] I have updated the relevant documentation.

### CODE REVIEW

File: tests/kubeless-test-client/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
-FROM debian:stable-slim
+FROM alpine:latest
 
 LABEL source="git@github.com:kyma-project/kyma.git"
 
-RUN apt-get update && apt-get install -y curl unzip && rm -r /var/lib/apt/lists/*
+RUN apk add --no-cache curl jq
 
-RUN curl -Lo /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/linux/amd64/kubectl && chmod +x /usr/local/bin/kubectl
+RUN curl -Lo /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x /usr/bin/kubectl
```
Comment (sayanh) [inline tests/kubeless-test-client/Dockerfile]: IMO versions should be explicit in order to keep the tests deterministic

File: tests/kubeless-test-client/Dockerfile [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-RUN curl -Lo /tmp/kubeless.zip https://github.com/kubeless/kubeless/releases/download/v1.0.0-alpha.2/kubeless_linux-amd64.zip && unzip -uq /tmp/kubeless.zip -d /tmp/ && mv /tmp/bundles/kubeless_linux-amd64/kubeless /usr/local/bin/ && rm -r /tmp/kubeless.zip /tmp/bundles && chmod +x /usr/local/bin/kubeless
+RUN curl -Lo /tmp/kubeless.zip "$(curl -s https://api.github.com/repos/kubeless/kubeless/releases/latest | jq -r '.assets[]|select(.name=="kubeless_linux-amd64.zip").browser_download_url')" && unzip -q /tmp/kubeless.zip -d /tmp/ && mv /tmp/bundles/kubeless_linux-amd64/kubeless /usr/bin/ && rm -r /tmp/kubeless.zip /tmp/bundles && chmod +x /usr/bin/kubeless
```
Comment (sayanh) [inline tests/kubeless-test-client/Dockerfile]: IMO versions should be explicit in order to keep the tests deterministic

File: tests/test-logging-monitoring/Dockerfile [hunk 3]
```diff
@@ -1,10 +1,10 @@
-FROM debian:stable-slim
+FROM alpine:latest
 
 LABEL source="git@github.com:kyma-project/kyma.git"
 
-RUN apt-get update && apt-get install -y curl unzip && rm -r /var/lib/apt/lists/*
+RUN apk add --no-cache curl
 
-RUN curl -Lo /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/linux/amd64/kubectl && chmod +x /usr/local/bin/kubectl
+RUN curl -Lo /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x /usr/bin/kubectl
```
Comment (sayanh) [inline tests/test-logging-monitoring/Dockerfile]: IMO versions should be explicit in order to keep the tests deterministic

File: tests/kubeless-test-client/Dockerfile (hunk 1 above)
Comment (gchbg) [reply tests/kubeless-test-client/Dockerfile]: Changed.

File: tests/kubeless-test-client/Dockerfile (hunk 2 above)
Comment (gchbg) [reply tests/kubeless-test-client/Dockerfile]: Changed.

File: tests/test-logging-monitoring/Dockerfile (hunk 3 above)
Comment (gchbg) [reply tests/test-logging-monitoring/Dockerfile]: Changed.

File: tests/kubeless-test-client/test-kubeless.go [hunk 4]
```diff
@@ -158,7 +158,7 @@ func main() {
 	}
 
 	log.Printf("Creating a new environment: %v", environment)
-	cmd := exec.Command("kubectl", "create", "ns", environment, "label", "env=true")
+	cmd := exec.Command("kubectl", "create", "ns", environment)
```
[the remark this answers is not in the document]
Comment (gchbg) [reply tests/kubeless-test-client/test-kubeless.go]: It cannot be set in the `create ns` command and is set several lines later. This is wrong syntax and fails the kubeless test immediately. Somehow master was broken.

Besides, this has been rewritten entirely in my other PR which got merged in the meantime.

**Pull request merged**